basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,236 @@
+
+it('tests if values are less than 90', () => {
+    const ninetyLess = helpers.lessThanNinety;
+
+    const Values1 = ninetyLess('50');
+    const Values2 = ninetyLess('200');
+
+    expect(Values1).toEqual(true);
+    expect(Values2).toEqual(false);
+});
+
+it('tests if values are more than 50', () => {
+    const fiftyMore = helpers.greaterThanFifty;
+
+    const Values1 = fiftyMore('40');
+    const Values2 = fiftyMore('200');
+
+    expect(Values1).toEqual(false);
+    expect(Values2).toEqual(true);
+});
+
+it('Add two numbers', () => {
+    const addTwoNumbers = helpers.add;
+
+    const value1 = addTwoNumbers(2, 3);
+    const value2 = addTwoNumbers(-3, 3);
+
+    expect(value1).toBe(5);
+    expect(value2).toBe(0);
+});
+
+it('minus 2 numbers; return sum', () => {
+    const subtract = helpers.subtract;
+
+    const one = subtract(4, 3);
+    const five = subtract(10, 5);
+
+    expect(one).toBe(1);
+    expect(five).toBe(5);
+});
+
+
+it('divide 2 numbers; return sum', () => {
+    const divide = helpers.divide;
+
+    const four = divide(12, 3);
+    const two = divide(10, 5);
+
+    expect(four).toBe(4);
+    expect(two).toBe(2);
+});
+
+
+it('multiply 2 numbers; return sum', () => {
+    const multiply = helpers.multiply;
+
+    const six = multiply(2, 3);
+    const ten = multiply(2, 5);
+
+    expect(six).toBe(6);
+    expect(ten).toBe(10);
+});
+
+it('get remainder of 2 numbers', () => {
+    const getRemainder = helpers.getRemainder;
+
+    const value1 = getRemainder(10, 3);
+    const value2 = getRemainder(21, 4);
+
+    expect(value1).toBe(1);
+    expect(value2).toBe(1);
+});
+
+it('checks for even number', () => {
+    const isEven = helpers.isEven;
+
+    const Value1 = isEven('66');
+    const Value2 = isEven('333');
+
+    expect(Value1).toEqual(true);
+    expect(Value2).toEqual(false);
+});
+
+it('checks for odd number', () => {
+    const isOdd = helpers.isOdd;
+
+    const Value1 = isOdd('66');
+    const Value2 = isOdd('333');
+
+    expect(Value1).toEqual(false);
+    expect(Value2).toEqual(true);
+});
+
+it('checks for square number', () => {
+    const square = helpers.square;
+
+    const eightyOne = square(9, 9);
+    const four = square(2, 2);
+
+    expect(eightyOne).toBe(81);
+    expect(four).toBe(4);
+});
+
+it('checks for cubed number', () => {
+    const cubed = helpers.cube;
+
+    const twentySeven = cubed(3, 3, 3);
+    const eight = cubed(2, 2, 2);
+
+    expect(twentySeven).toBe(27);
+    expect(eight).toBe(8);
+});
+
+it('checks for raise to power', () => {
+    const raiseToPower = helpers.raiseToPower;
+
+    const Power = raiseToPower(3, 5);
+
+    expect(Power).toBe(243);
+});
+
+it('checks for round number', () => {
+    const roundNumber = helpers.roundNumber;
+
+    const round = roundNumber(3.3);
+
+    expect(round).toBe(3);
+});
+
+
+it('checks for roundup number', () => {
+    const roundUp = helpers.roundUp;
+
+    const upRound = roundUp(1.95);
+
+    expect(upRound).toBe(2);
+});
+
```

Comment:
I think for the previous 5 tests, maybe you could make the names more specific. Because it's not really checking for a square, it's squaring the number(s) provided, or cubing them, or rounding them up or down.